README.md

```diff
@@ -49,16 +60,17 @@ Issues: #3
 
 ### The data file
 
-The core data file is encoded in [csv](http://tools.ietf.org/html/rfc4180) using the [UTF-8](http://en.wikipedia.org/wiki/UTF-8) character encoding. This file must have a header, i.e. the first row
-must contain the list of column names. While the file may contain any number of columns, columns with a specific 
-meaning in our context are detected by name:
+The core data file is encoded in [csv](http://tools.ietf.org/html/rfc4180) using the [UTF-8](http://en.wikipedia.org/wiki/UTF-8) character encoding. This file must have a header, i.e. the first row must contain the list of column names. While the file may contain any number of columns, columns with a specific meaning in our context are detected by name:
 
 - `ID`: identifies a row in the data file; either a local ID - preferably an [UUID](http://en.wikipedia.org/wiki/Universally_unique_identifier) - or an (equally universally unique) URL like http://wold.clld.org/word/7214142329897819 or http://wals.info/valuesets/1A-niv
 - `Language_ID`: identifies the language or variety the data in the row is about. A [Glottolog languoid URL](http://glottolog.org), or *glottocode* or ISO-639-3 code (FIXME: require a URL, or a disambiguating prefix?), or a local identifier.
+- `Feature_ID`: the `question` to be asked to the language, i.e. the `Domain` or `Function` or `Trait` to be investigated
```

Comment:
My initial proposal was to specify a family of formats (for separate data types), derived from a core; i.e. [wordlists](https://github.com/glottobank/cldf/blob/master/wordlist.md) may have column `Concept_ID`, while structural datasets may have a column `Feature_ID`. I think this would make the spec less abstract, but maybe there are generic applications which would work with lexical and structural data at the same time?